### DISCUSSION TITLE
Simplify enum access

### DISCUSSION THREAD

**Discussion by mikasoukhov** [dotnet/csharplang#5391] [Language Ideas]
Now

`myObj.EnumField1 = EnumFields1.Value1;`

Can be

`myObj.EnumField1 = _.Value1;`

**Comment by 333fred** [comment]
See https://github.com/dotnet/csharplang/issues/2926, which would subsume this.

**Comment by The-Futurist** [comment]
> Now
> 
> `myObj.EnumField1 = EnumFields1.Value1;`
> 
> Can be
> 
> `myObj.EnumField1 = _.Value1;`

`using static` works with enums by the way:

![image](https://user-images.githubusercontent.com/12262952/141156297-ee259650-476b-4049-abdc-68d963bf5282.png)
